core/storage/predefined_keys.hpp

```diff
@@ -40,8 +40,7 @@ namespace kagome::storage {
 
   inline const common::Buffer kWarpSyncOp = ":kagome:WarpSync:op"_buf;
 
-  inline const common::Buffer kBabeConfigRepositoryImplGenesisSlot =
-      ":kagome:BabeConfigRepositoryImpl:genesis_slot"_buf;
+  inline const common::Buffer kFirstBlockSlot = ":kagome:first_block_slot"_buf;
```

Comment:
Will this key be reused by sassafras with same value encoding?
Old name/key was specific

Reply:
Yes